src/app/product/index.ts

```diff
@@ -2,7 +2,7 @@ import { NgModule } from '@angular/core';
 import { SharedModule } from './../shared/index';
 
 import { RouterModule } from '@angular/router';
-
+import { ProgressbarModule } from 'ngx-bootstrap';
```

Comment:
Don't use this please 😠, as it will pull everything in build.
use `import { ProgressbarModule } from 'ngx-bootstrap/progressbar';`